src/NoteMouseHandler/PencilMouseHandler.js

```diff
@@ -1,17 +1,18 @@
 import _ from "lodash"
 import SharedService from "../services/SharedService"
-import MouseHandler, { defaultActionFactory, getLocal } from "./NoteMouseHandler"
+import MouseHandler, { defaultActionFactory } from "./NoteMouseHandler"
```

Comment:
__[jshint]__
*  'import' is only available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->